integration-tests/plugins/__tests__/price-lists/admin/add-price-list-price-batch.spec.ts

```diff
@@ -90,8 +90,8 @@ describe("POST /admin/price-lists/:id/prices/batch", () => {
       {
         title: "test price list",
         description: "test",
-        ends_at: new Date(),
-        starts_at: new Date(),
+        ends_at: new Date().toISOString(),
+        starts_at: new Date().toISOString(),
```

Comment:
the service is typed as string right now, which creates an inconsistency with the rest-api and module data-layer that's not necessarily wanted. I've created a pr to convert these inputs to date here, which would allow us to remove the `toIsoString` from the module invocation and instead allow the repositories to handle any conversions if required: https://github.com/medusajs/medusa/pull/5768

(feel free to drop a review and weigh in on the conversation 😉 )